src/components/App.js

```diff
@@ -7,30 +7,38 @@ import FavDetails from '../pages/FavDetails';
 import Login from '../pages/Login';
 import Header from '../components/Header';
 import useAxiosFetch  from '../utils/hooks/UseAxiosFetch';
+import { ThemeContext } from "../Context/ThemeContext";
+import { useQuery } from '../Context/QueryContext';
+// import { UserContext } from "../Context/UserContext";
+import './App.css';
+
 
 const API_URL = `https://www.googleapis.com/youtube/v3/search?part=snippet&type=video&maxResults=5&key=${process.env.REACT_APP_UNSPLASHED_KEY}&q=`;
 
 function App() {
+  
+  const theme = useContext(ThemeContext);
+  const { state }  = theme ? theme : {};
+  const { darkMode } = state ? state : {};
```

Comment:
You can make this simple
```suggestion
  const { darkMode } = theme?.state ?? {};
```